apps/costcontrol/js/app.js

```diff
@@ -378,31 +378,25 @@ var CostControlApp = (function() {
           window.location.hash = hashFromLastRun || newHash;
           hashFromLastRun = null;
 
-          // XXX: Break initialization to allow Gecko to render the animation on
-          // time.
-          setTimeout(function continueLoading() {
```

Comment:
This timeout is intended to allow the animation to work. It should be implemented with a `requestAnimationFrame()` call but not removed. Please, restore it.